plasmapy/particles/tests/test_atomic.py

```diff
@@ -483,18 +483,14 @@ def test_isotopic_abundance() -> None:
         isotopic_abundance("Og-2")
 
 
-isotopic_abundance_elements = (
-    atomic_number(atomic_numb) for atomic_numb in range(1, 119)
-)
```

Comment:
I seem to have discovered Python generators in Dec 2017 and perhaps maybe had just a little bit too much fun with them...